src/utils/api.ts

```diff
@@ -0,0 +1,5 @@
+const API_ENDPOINT = 'https://meek-gumption-c04779.netlify.app/vacations';
+
+type APIPart = 'reviews' | 'recommendations' | 'rooms';
```

Comment:
It would be good if the delay was passed in via a query param too. Otherwise it's not clear where the delay is added